frontend/app/mocks/raoidc.server.ts

```diff
@@ -145,7 +145,7 @@ async function generateUserInfoToken(clientPublicKey: string, serverPrivateKey:
     locale: 'en-CA',
     sid: '00000000-0000-0000-0000-000000000000',
     sin: '800011819',
-    sub: '00000000-0000-0000-0000-000000000000',
+    sub: '76c48130-e1d4-4c2f-8dd0-1c17f9bbb4f6',
```

Comment:
This is not really future-proof. Please add a TODO that says something like "allow caller to specify sub as a parameter"

Reply:
todo added